### ISSUE TITLE
.jshintrc not being generated

### ISSUE THREAD

**Issue by fseminario** [romainberger/yeoman-wordpress#55]
Mac OS X 10.10.2 
Node: v0.12.2
Npm: 2.7.4
Yo (readme code should be updated to "yo --version"): 1.4.6

![screen shot 2015-04-24 at 5 17 32 pm](https://cloud.githubusercontent.com/assets/4037582/7330580/d8c0e35c-eaa5-11e4-9bd3-fbcd03171644.png)

I run npm and bower install but it hangs at this

![screen shot 2015-04-24 at 5 24 51 pm](https://cloud.githubusercontent.com/assets/4037582/7330629/d973f284-eaa6-11e4-9475-337fab732012.png)

**Comment by dnzmsc** [comment]
@fseminario Hi, did you solve this issue?

**Comment by fseminario** [comment]
@dnzmsc nope. Awaiting answer from community. Haven't been able to use the generator.
